src/Features/ScreenSpaceShadows.cpp

```diff
@@ -249,9 +258,13 @@ void ScreenSpaceShadows::ModifyLighting(const RE::BSShader*, const uint32_t)
 
 					data.RcpBufferDim.x = 1.0f / data.BufferDim.x;
 					data.RcpBufferDim.y = 1.0f / data.BufferDim.y;
+					if (REL::Module::IsVR())
+						data.ProjMatrix = shadowState->GetVRRuntimeData().cameraData.getEye().projMat;
+					else
+						data.ProjMatrix = shadowState->GetRuntimeData().cameraData.getEye().projMat;
 
-					data.ProjMatrix = shadowState->m_CameraData.m_ProjMat;
-					data.InvProjMatrix = XMMatrixInverse(nullptr, shadowState->m_CameraData.m_ProjMat);
+					data.ProjMatrix = shadowState->GetRuntimeData2().cameraData.getEye().projMat;
```

Comment:
@doodlum I'm not sure why this line exists when I merge but it's redundant and should be removed. It's sit by the if else above.

Reply:
Nevermind found it. Had a debug statement that I didn't push and apparently Git thought it was related to this.